apps/db/priv/repo/migrations/20181010105152_create_video_captions.exs

```diff
@@ -0,0 +1,16 @@
+defmodule DB.Repo.Migrations.CreateVideoCaptions do
+  use Ecto.Migration
+
+  def change do
+    create table(:videos_captions) do
+      add(:video_id, references(:videos, on_delete: :delete_all))
+      add(:content, :text, null: false)
+      add(:format, :string, null: false)
+      add(:locale, :string, null: false)
```

Comment:
This uses `locale` but the video uses `language`. If there is no difference in meaning, `language` might be better for consistency